packages/relay/test/counter.test.ts

```diff
@@ -111,11 +110,16 @@ describe('GET /counter', function () {
     // put this test in the end since this test will
     // go to next epoch
     it('should delete the EpochKeyAction table after the epoch ended', async function () {
-        const epoch = sync.calcCurrentEpoch()
+        const epoch = await sync.loadCurrentEpoch()
         const epochRemainingTime = sync.calcEpochRemainingTime()
+
+        await ethers.provider.send('evm_increaseTime', [
+            epochRemainingTime + 100000,
+        ])
+
         // add 10s to make sure this epoch ended
```

Comment:
modify this comment and move before
```js
await ethers.provider.send('evm_increaseTime', [
            epochRemainingTime + 100000,
        ])
```
you used ethers to fast forward block, then you can remove unnecessary package
`@nomicfoundation/hardhat-network-helpers`